### DISCUSSION TITLE
Primefaces panelgrid with layout="tabular" columns="0" and selectOneMenu with selectItems + <p:column> adds an additional <td> in the selectOneMenu

### DISCUSSION THREAD

**Discussion by ItsPeeeZn** [primefaces/community#1565] [PrimeFaces] [answered]
Hello Primefaces Team and users!

I found a bug in primefaces 13.0.7 it seems like the renderer is making some troubles when using a selectOneMenu in a panelgrid that is set to tabular.

To make it easier to reproduce I used the Java classes provided in the showcase for the selectOneMenu. So it´s pretty easy :) I did not know if this is a known issue or not, I searched for it but could not find anything related. 

So this is the XHTML Code. 
![image](https://github.com/primefaces/community/assets/137711395/fd3d2b48-5e7b-4563-8ed3-6d885ac7ad52)

and that´s the result:
![image](https://github.com/primefaces/community/assets/137711395/3910ae38-a085-49a4-a255-f44c5929e24f)

as soon as you switch to another layout type like grid
![image](https://github.com/primefaces/community/assets/137711395/8776873c-48e7-44df-8dd3-7783ac9d647d)

that´s the result:
![image](https://github.com/primefaces/community/assets/137711395/12710fb8-c5fa-45e9-86cc-e63e0fb1c455)

as stated the selectOneMenu implementation is the one from the showcase :)
![image](https://github.com/primefaces/community/assets/137711395/412311e5-2eac-4678-9ffe-528d47339cbb)

I do not know if I should create a Bug Report on GitHub because it is stated first go to the forum :)
Thank you all!

**Comment by FlipWarthog** [comment]
Can you dump this into a [primefaces-test](https://github.com/primefaces/primefaces-test) reproducer? It'll make it even easier for others to take a look and if it ends up being an issue needing a ticket, the reproducer can be attached right to it.

**Reply by ItsPeeeZn** [reply]
Yes sure, I can do that tomorrow! :)

**Reply by melloware** [reply]
Any luck with the reproducer?

**Reply by ItsPeeeZn** [reply]
So far I did not had any time to dump something in the primefaces-test reproducer because i was getting sick the next day xD - should I still do that? or anyone else had time to check?

**Reply by melloware** [reply]
Nope still waiting on you.

**Reply by melloware** [reply, marked as answer]
I will close this discussion until a reproducer is provided
